src/Symfony/Bundle/FrameworkBundle/Tests/Command/CachePoolClearCommandTest.php

```diff
@@ -35,6 +35,10 @@ protected function setUp(): void
      */
     public function testComplete(array $input, array $expectedSuggestions)
     {
+        if (!class_exists(CommandCompletionTester::class)) {
+            $this->markTestSkipped('Test command completion requires symfony/console 5.4+.');
+        }
```

Comment:
FrameworkBundle already defined `"symfony/console": "^5.4|^6.0"` in its composer.json. This check should not be necessary.